### ISSUE TITLE
add mail field to Participant

### ISSUE THREAD

**Issue by phtyson** [yawlfoundation/yawl#654]
In my use cases, which make extensive use of MailService, it is necessary to have email field available from the Participant object.

I believe this would be a generally useful improvement to YAWL, and would not interfere with existing org data that doesn't need email field on participants.

I have added this capability in my fork, for YAWL 4.x (using resourceService administration) and 5.x (using yawlui).

Before submitting patch or PR, I want to request comments from the community on this feature.

**Comment by ahense** [comment]
Yes, I would really encourage you to submit this extension. I have always seen the necessity to add e-mail information to participants. One useful feature would be to be able to notify the non-frequent users when a work item has been assigned to them. At a later stage users could configure in which cases they want to be notified by the system.

**Comment by phtyson** [comment]
Implemented with commit 6778e7d6c9596257e57fa8da6c36391bd86161c0.
